test/ordered_set_test.dart

```diff
@@ -320,14 +320,24 @@ void main() {
 
       test('sorts after remove', () {
         final orderedSet = OrderedSet<int>();
-        // The initial elements must be in order.
+        // The initial elements must be in order to reproduce the issue.
         orderedSet.addAll([1, 3, 4]);
         expect(orderedSet.toList().join(), '134');
         expect(orderedSet.remove(4), true);
         expect(orderedSet.toList().join(), '13');
         expect(orderedSet.add(2), true);
         expect(orderedSet.toList().join(), '123');
       });
+
+      test('correct order after remove', () {
+        final set = OrderedSet<int>();
```

Comment:
should we call it orderedSet to be consistent?
`set` trips github highlighting

Reply:
Fixed